worker/src/main/java/org/verapdf/webapp/worker/handler/QueueSenderErrorEventHandlerImpl.java

```diff
@@ -0,0 +1,28 @@
+package org.verapdf.webapp.worker.handler;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.verapdf.webapp.queueclient.entity.QueueErrorEventType;
+import org.verapdf.webapp.queueclient.entity.SendingToQueueErrorData;
+import org.verapdf.webapp.queueclient.handler.QueueSenderErrorEventHandler;
+
+@Service
+public class QueueSenderErrorEventHandlerImpl implements QueueSenderErrorEventHandler {
+
+	private static final Logger LOGGER
+			= LoggerFactory.getLogger(QueueSenderErrorEventHandlerImpl.class);
+
+	@Override
+	public void handleEvent(SendingToQueueErrorData event) {
+		if(event.getQueueErrorEventType().equals(QueueErrorEventType.SENDING_ERROR_CALLBACK)) {
+			LOGGER.error("Message: " + event.getMessage()
```

Comment:
as this is a logger case - we can use placeholders to avoid string concatenation (`{}`)